scripts/partner_batch_imports.py

```diff
@@ -87,7 +87,7 @@
 
 
 class Biblio:
-    ACTIVE_FIELDS = [
+    ACTIVE_FIELDS: tuple[str, ...] = (
```

Comment:
We can drop the type hint here and elsewhere for these tuples; it should be automatically induced by mypy, and doesn't add too much to the developer experience to explicitly write it out.

```suggestion
    ACTIVE_FIELDS = (
```

Reply:
I have made the changes, and there are no errors or issues now .Thank you for your guidance.